### ISSUE TITLE
Flag '--cpuset' doesn't exist

### ISSUE THREAD

**Issue by sege** [garethr/garethr-docker#448]
Flag '--cpuset' doesn't exist with Docker 1.10.
Output when trying to start: "
Starting docker-elasticsearch:	flag provided but not defined: --cpuset"

From init-script: "
    printf "Starting $prog:\t"
                $docker rm  elasticsearch >/dev/null 2>&1
            $docker run \
    --net bridge -m 0b --cpuset=0-7
..<snip>"

Hiera-conf: "
docker::run_instance::instance:
  elasticsearch:
    image: 'elasticsearch:2.2'
    cpuset: '0-7'
...<snip>"

I think it should be '--cpuset-cpus' according to the docs.

**Comment by petems** [comment]
Yep, looks like it was deprecated: https://docs.docker.com/engine/deprecated/

```
The following double-dash options are deprecated and have no replacement:

docker run --cpuset
```

Should be a quick fix, let me PR :smile:

**Comment by rogierslag** [comment]
Can I submit a PR for this new behaviour?
